src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,23 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static LocalDate CURRENT_DATE = LocalDate.now();
```

Comment:
remove this field and use local variables instead

Reply:
Done